src/main/java/net/neoforged/moddevgradle/dsl/UnitTest.java

```diff
@@ -30,4 +31,12 @@ public void enable() {
      * will be added to that mod at runtime.
      */
     public abstract Property<ModModel> getTestedMod();
+
+    /**
+     * The mods to load when running unit tests. Defaults to all mods registered in the project.
+     * This should include {@link #getTestedMod()}.
```

Comment:
Should, otherwise setting `testedMod` is useless.

Reply:
It will now fail with an error that looks like this:
```
A problem occurred configuring project ':jijtest'.
> Failed to query the value of property 'modFolders'.
   > The tested mod (jijtest) must be included in the mods loaded for unit testing ([coremod]).
```